mock/mock.spec

```diff
@@ -177,6 +177,9 @@ cp -a docs/mock.cheat %{buildroot}%{_datadir}/cheat/mock
 install -d %{buildroot}/var/lib/mock
 install -d %{buildroot}/var/cache/mock
 
+mkdir -p %buildroot/%_pkgdocdir
```

Comment:
I would write `%{buildroot}` and `%{_pkgdocdir}`, to make the specfile look consistent.

The slash before `%_pkgdocdir` and `%{_sysconfdir}` is redudundant?

Reply:
Fixed, the additional slash was ugly, good catch.  Thanks for the review!

I dislike using the `%{macro}` variant when it is not necessary, same as
with shell variables (the `${prefix}name` case).  But changed, FWIW.